app/frontend/src/features/search/users.ts

```diff
@@ -68,38 +60,39 @@ export const layers: Record<LayerKeys, AnyLayer> = {
     id: "unclustered-points",
     layout: {
       "icon-image": "user-pin",
-      "icon-allow-overlap": true,
-    },
-    source: "clustered-users",
-    type: "symbol",
-  },
-
-  users: {
-    filter: ["!", ["has", "point_count"]],
-    id: "users",
-    layout: {
-      "icon-image": "user-pin",
+      "icon-anchor": "bottom",
       "icon-allow-overlap": true,
     },
     paint: {
-      "icon-opacity": [
+      "icon-color": [
+        "case",
+        ["boolean", ["feature-state", "selected"], false],
+        theme.palette.primary.main,
+        theme.palette.grey[500],
+      ],
+      "icon-halo-width": 2,
+      "icon-halo-color": [
         "case",
         ["boolean", ["feature-state", "selected"], false],
-        1,
-        0.5,
+        theme.palette.primary.main,
+        theme.palette.grey[500],
       ],
+      "icon-halo-blur": 2,
     },
-    source: "all-objects",
+    source: "clustered-users",
     type: "symbol",
   },
 };
 
 const addPinImages = (map: MaplibreMap) => {
+  if (map.hasImage("user-pin")) return;
```

Comment:
I'm guessing leaving this out would work, but ends up doing a request to get the image still if later on it's found to be loaded already?

What was happening before with the old code? 🤔

Reply:
I was getting console warnings about the image already being added to the map. 
It's weird because this function is only called once as far as I could see.